build-config/src/main/resources/forbidden-common.txt

```diff
@@ -0,0 +1,115 @@
+# Hibernate Validator, declare and validate application constraints
+#
+# License: Apache License, Version 2.0
+#  See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+
+# This file contains forbidden API signatures common to all modules
+
+@ignoreUnresolvable
+@defaultMessage Shouldn't be used
+
+################################################################################################################
+# Some packages that should be used
+java.awt.**
+sun.**
+org.slf4j.**
+junit.framework.**
+
+################################################################################################################
+# org.testng.Assert#fail() should be used
+org.assertj.core.api.Fail#fail()
+org.testng.FileAssert#fail()
+org.assertj.core.api.Assertions#fail()
+
+################################################################################################################
+# Nobody should be using StringBuffer anymore
+java.lang.StringBuffer
```

Comment:
ahh.. just remembered that one of the plugins (either forbidden-apis or check-style illegal imports) wasn't picking up the `java.lang.StringBuffer` as it's from the "automatically imported package" ... But not sure about which one it was of these two  ... 
Again - I'll play with different cases and I'll let you what works and what's not :)